static/manifest.json

```diff
@@ -59,7 +59,8 @@
     "commons.contentscript-inpage.js",
     "commons.popup-inpage.js",
     "inpage.js",
-    "images/logo.png"
+    "images/logo.png",
+    "modal.css"
```

Comment:
can you move `modal.css` to  `css/modal.css`?

Reply:
done.